Automattic/jetpack
# Masterbar: Correctly unfold top level menu items for submenu items

For wp-admin to know when to unfold a top-level menu item, we need to give it their new slugs where we changed them.
This PR standardizes that to using closures with the `parent_file` filter since the function body can just be a ternary.
Not sure about Jetpack's stance on closures but I've seen them used elsewhere, please let me know if the use here gives cause for concern.

#### Changes proposed in this Pull Request:

* Added `parent_file` callbacks to all menu items where we changed the menu slug of the top-level menu item.
* Moved existing `parent_file` callbacks into becoming closures for consistency

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* In wp-admin, select Posts > Categories or Posts > Tags or Settings > Ratings (if exists) and see if the top-level menu item is unfolded when the page is loaded.

#### Proposed changelog entry for your changes:


None needed.
